src/libsync/logger.cpp

```diff
@@ -89,6 +89,30 @@ bool Logger::isLoggingToFile() const
 void Logger::doLog(QtMsgType type, const QMessageLogContext &ctx, const QString &message)
 {
     const QString msg = qFormatLogMessage(type, ctx, message);
+#if defined(Q_OS_WIN) && defined(QT_DEBUG)
+    // write logs to Output window of Visual Studio
+    {
+        QString prefix;
+        switch (type) {
+        case QtDebugMsg:
+            break;
+        case QtInfoMsg:
+            break;
+        case QtWarningMsg:
+            prefix = "[WARNING] ";
```

Comment:
@Dimon4eg Would be nice to have every string literal wrapped with `QStringLiteral()`. Otherwise, looks good to go.

Reply:
done